CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Ben Asher](https://github.com/benasher44)
   [#5860](https://github.com/CocoaPods/CocoaPods/issues/5860)
 
+* Fix linting of private pods when using libraries.
```

Comment:
To ensure that the changelog is generated correctly, we need two spaces at the end of this, i.e `* Fix linting of private pods when using libraries.  `